tests/informatics/test_predictors.py

```diff
@@ -24,6 +25,17 @@ def simple_predictor() -> SimpleMLPredictor:
                              training_data=AraTableDataSource(uuid.UUID('e5c51369-8e71-4ec6-b027-1f92bdc14762'), 0))
 
 
+@pytest.fixture
+def molecule_featurizer() -> MolecularStructureFeaturizer:
+    return MolecularStructureFeaturizer(
+        name="Molecule featurizer",
+        description="description",
+        descriptor=MolecularStructureDescriptor("SMILES"),
+        features=["standard"],
+        excludes=[]
```

Comment:
If this is for a test, why aren't we explicitly excluding one of the features to make sure we're exercising all code?

Reply:
Could do - we don't _actually_ test the logic here, though. We don't do set math or anything in the client. I can confirm a non-empty list is preserved, though.